.evergreen/config.yml

```diff
@@ -717,6 +718,28 @@ functions:
         export RUN_MONGOHOUSE_TESTS=true
         ./src/libmongoc/test-libmongoc --no-fork -l /mongohouse/* -d
         unset RUN_MONGOHOUSE_TESTS
+  test versioned api:
+  - command: shell.exec
+    type: test
+    params:
+      working_dir: mongoc
+      shell: bash
+      script: |-
+        set -o errexit
+        set -o xtrace
+        export COMPRESSORS='${COMPRESSORS}'
+        export CC='${CC}'
+        export AUTH=${AUTH}
+        export SSL=${SSL}
+        export URI=${URI}
+        export IPV4_ONLY=${IPV4_ONLY}
+        export VALGRIND=${VALGRIND}
+        export MONGOC_TEST_URI=${URI}
+        export DNS=${DNS}
+        export ASAN=${ASAN}
+        export MONGODB_API_VERSION=1
```

Comment:
It was unclear to me which of these env variables were necessary for the versioned API tests; let me know if any are not needed.

Reply:
Gotcha, thanks. For now, I'll leave these env vars in, as we may end up running all integration tests in this function.

Reply:
Sounds good!